src/components/labelled-date-input/LabelledDateInput.css

```diff
@@ -0,0 +1,12 @@
+.form-control {
```

Comment:
It's not good to override some styles from library. Here we can at least explicitly specify where this class appears on the markup. We can explicitly pass desired className to the DateTime component, using `inputProps` prop. [Read more about this prop](https://github.com/YouCanBookMe/react-datetime#customize-the-input-appearance). Let it be `labelled-date-input__input`. And we can apply this styles to `labelled-date-input__input` here instead of `form-control`.

Just because `form-control` is too broad selector. It's difficult to understand where this class is used in the markup. And when you explicitly write, that DateTime component's input element has this class, it's much better